src/bounded_encoded_length.rs

```diff
@@ -78,3 +78,25 @@ impl<T: BoundedEncodedLen, const N: usize> BoundedEncodedLen for [T; N] {
 		T::max_encoded_len().saturating_mul(N)
 	}
 }
+
+impl<T: BoundedEncodedLen> BoundedEncodedLen for Option<T> {
+	fn max_encoded_len() -> usize {
+		1 + T::max_encoded_len()
```

Comment:
everything should be saturating i think

Reply:
Oops, right. https://github.com/paritytech/parity-scale-codec/pull/267/commits/5f886b45cebb31106c357ecc33a86b492c476a46